evap/static/js/csrf_utils.js

```diff
@@ -0,0 +1,32 @@
+// based on: https://www.djangoproject.com/weblog/2011/feb/08/security/
+// csrf setup
+function getCookie(name) {
+	var cookieValue = null;
+	if (document.cookie && document.cookie != '') {
+		var cookies = document.cookie.split(';');
+		for (var i = 0; i < cookies.length; i++) {
+			var cookie = jQuery.trim(cookies[i]);
+			// Does this cookie string begin with the name we want?
+			if (cookie.substring(0, name.length + 1) == (name + '=')) {
+				cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+				break;
+			}
+		}
+	}
+	return cookieValue;
```

Comment:
you could just `return decodeURIFoo()` inside the loop.

also, is it really that complicated to retrieve some cookie value? :) (i honestly don't know)